tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/compat/v1/V1.scala

```diff
@@ -0,0 +1,17 @@
+package me.shadaj.scalapy.tensorflow.compat.v1
+
+import me.shadaj.scalapy.py
+import me.shadaj.scalapy.tensorflow.Tensor
+
+@py.native trait V1 extends py.Object {
```

Comment:
Hmm it's still a bit unclear whether a given trait corresponds to a package or to a class :thinking:

Reply:
Unless introducing this distinction is meant for another PR

Reply:
yup thats another PR 
I haven't figured it out yet